airbyte-server/src/test/java/io/airbyte/server/converters/SchemaConverterTest.java

```diff
@@ -69,6 +69,7 @@ void convertToApiSchema() {
   @Test
```

Comment:
need to add existing unit tests here to check that the new fields are properly mapped.